c++/src/capnp/rpc-twoparty.h

```diff
@@ -84,9 +91,13 @@ class TwoPartyVatNetwork: public TwoPartyVatNetworkBase,
   // to be sent on this connection. This may be useful for backpressure.
 
   size_t getCurrentQueueCount() { return currentQueueCount; }
-  // Get the count outgoing messages that are currently queued in memory waiting
+  // Get the count of outgoing messages that are currently queued in memory waiting
   // to be sent on this connection. This may be useful for backpressure.
 
+  kj::Duration getOutgoingMessageWaitTime();
```

Comment:
Updated the method name and comment here to reflect reality -- we aren't measuring the age of the oldest outgoing message, but how long it's been waiting to be written.